### ISSUE TITLE
Site search a11y accessibility issues 

### ISSUE THREAD

**Issue by Bosch-0** [BitcoinDesign/Guide#844]
Ran the a11y Netlify plugin on bitcoin.design which tests the site against a11y accessibility standards and received 3 errors all relating to the sites search, no others popped up. 

```
12:25:13 PM: [Search | Bitcoin Design (_site/search.html)] error 1: This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid. (<form action="/search.html" msethod="get">
12:25:13 PM:   <input type="text" id="searc...</form>)
[Search | Bitcoin Design (_site/search.html)] error 1: This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined. (<input type="text" id="search-page-input" name="query">)
12:25:13 PM: [Search | Bitcoin Design (_site/search.html)] error 1: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate. (<input type="text" id="search-page-input" name="query">)
```

**Comment by Bosch-0** [comment]
Closing in favour of https://github.com/BitcoinDesign/Guide/issues/846
